ev3dev/core.py

```diff
@@ -2351,6 +2353,10 @@ def color(self):
         return self.value(0)
 
     @property
+    def color_name(self):
+        return self.COLORS[self.color]
+
+    @property
     def raw(self):
         """
         Red, green, and blue components of the detected color, in the range 0-1020.
```

Comment:
Should we add a comment here pointing to the `calibrate_white` method?

Reply:
`raw()` doesn't do anything with it though...it only impacts `rgb()`

Reply:
Something like:

> The colors returned by this method may be unexpectedly dark. If this is an issue, check out the rgb() and calibrate_white() methods.

Reply:
They scale from 0-1020 though so if you displayed them as if they were 0-255 they would appear fairly bright (assuming none went over the 255 limit).  What I'm trying to say is whether they appear dark or not is really up to how the user is using the returned values.  If the user is scaling them much like what we do in rgb() then they wouldn't appear that dark.

Reply:
My issue is that the doc comment says the range is "0-1020" yet the values will never get up that high. I think that expectation should be laid out in the comment here and in the `rgb` method.

Reply:
ah I follow you now, adding this
```
+        Red, green, and blue components of the detected color, officially in the
+        range 0-1020 but the values returned will never be that high. We do not
+        yet know why the values returned are low, but pointing the color sensor
+        at a well lit sheet of white paper will return values in the 250-400 range.
+
+        If this is an issue, check out the rgb() and calibrate_white() methods.
```

Reply:
Perfect, I'm happy with this PR.